StackOverflow/Model/DataResult.h

```diff
@@ -0,0 +1,16 @@
+//
+//  DataResult.h
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface DataResult : NSObject
+
++ (NSArray *)resultFromJSON:(NSData *)objectNotation error:(NSError **)error;
```

Comment:
resultFromJSON is not a good name, a result is too generic. What are we actually getting from the JSON, questions, answers, groups ?